content/en/docs/Installing and Configuring DreamFactory/_index.md

```diff
@@ -211,6 +211,18 @@ Immediately following this prompt you'll be informed of successful setup:
 	* your instance using a browser. You can run "php artisan serve"
 	* to try out your instance without setting up a web server.
 	**********************************************************
+After creating an Admin, your instance may show the following error:
```

Comment:
Generally speaking,  I think this should go in its own h2 or h3 header. Perhaps a troubleshooting section. Or in a section for errors when upgrading to silver or gold.